ps_linklist.php

```diff
@@ -295,16 +299,21 @@ private function getRepository()
                         //Module is not installed so its services are not loaded
                         /** @var LegacyContext $context */
                         $legacyContext = $container->get('prestashop.adapter.legacy.context');
+
                         /** @var Context $shopContext */
                         $shopContext = $container->get('prestashop.adapter.shop.context');
+                        
+                        /** @var FeatureInterface $multiStoreFeature */
+                        $multiStoreFeature = $container->get('prestashop.adapter.feature.multistore');
+                        
                         $this->repository = new LinkBlockRepository(
```

Comment:
you could use the service definition here of instanciating the class directly (you did add the new parameters in the yaml service definition, let's use it !  or am I missing something ?

Reply:
you're right 🤔 it was done on purpose in previous version but right now we could just use service definition, I'll try that